src/components/HomePage.jsx

```diff
@@ -1,22 +1,65 @@
-import React, { Component } from 'react';
-import Image from "./assets/image.jpg";
+import React from 'react';
+import SearchIcon from "./assets/search.png";
```

Comment:
As I mentioned earlier, you need to get these icons from react-icons or from fontawesome libraries. It's not necessary to download and store them in the assets folder. Check here pls: https://react-icons.github.io/react-icons, and here: https://react-icons.github.io/react-icons/search?q=search